examples/quickstart/matrix_multiplication.cpp

```diff
@@ -0,0 +1,121 @@
+////////////////////////////////////////////////////////////////////////////////
+//  Copyright (c) 2021 Dimitra Karatza
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+////////////////////////////////////////////////////////////////////////////////
+
+// Parallel implementation of matrix multiplication
+
+#include <hpx/local/algorithm.hpp>
+#include <hpx/local/execution.hpp>
+#include <hpx/local/init.hpp>
+
+#include <iostream>
+#include <random>
+#include <vector>
+
+std::mt19937 gen;
+
+///////////////////////////////////////////////////////////////////////////////
+//[mul_print_matrix
+void print_matrix(std::vector<int> M, int rows, int cols, const char* message){
```

Comment:
```suggestion
void print_matrix(std::vector<int> M, std::size_t rows, std::size_t cols, const char* message){
```
I think the examples are not checked for formatting, but inspect will complain about the longer line length. If you haven't used the formatter before you'll need `clang-format` version 9 (can be installed with `clang-format-9` at least on Ubuntu), and then you run `clang-format -i path/to/file.cpp` to format it.